aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -130,6 +132,21 @@ The following examples use the <xref:Microsoft.AspNetCore.Server.Kestrel.Core> n
 using Microsoft.AspNetCore.Server.Kestrel.Core;
```

Comment:
I would move this below the json sample so it's closer to the other c# samples.

Reply:
They're going to need that for the example added. I'm expanding to include both the `Host` way (*Program.cs*) and the `Startup` way (*Startup.cs*). This also works ...

```csharp
public void ConfigureServices(IServiceCollection services)
{
    services.AddRazorPages();

    // Requires: using Microsoft.AspNetCore.Server.Kestrel.Core;
    // Configuration is an `IConfiguration` injected into the `Startup` class
    services.Configure<KestrelServerOptions>(Configuration.GetSection("Kestrel"));
}
```